docs/en/ingest-management/fleet/fleet-settings.asciidoc

```diff
@@ -0,0 +1,94 @@
+[[fleet-settings]]
+= {fleet} settings
+
+beta[]
+
+Configure {fleet} settings in {kib} to apply global settings to all {agent}s
+enrolled in {fleet}:
+
+. In {kib}, open the main menu, then click *Management > {fleet}*.
+
+. Click *{fleet} settings*
++
+--
+[cols="2*<a"]
+|===
+|
+[id="fleet-server-hosts-setting"]
+*{fleet-server} hosts*
+
+| The URLs your {agent}s will use to connect to a {fleet-server}. This setting
+is required. On self-managed clusters, you must specify one or more URLs. On
+{ecloud}, this field is populated automatically.
+
+//QUESTION: What is valid here? URL implies something like http://localhost:8220.
+//Is the scheme and port required or does the setting use defaults if their
+//not specified?
+
+//QUESTION: What do we need to say about IPv6 addresses?
```

Comment:
excellent question!  I don't know.  @ph @ruflin @mostlyjason ?

Reply:
I don't have time to wait before merging, so I close this issue and leave a todo item in the source files.